pac4j-core/src/test/java/org/pac4j/core/client/MockIndirectClient.java

```diff
@@ -38,7 +40,7 @@ public MockIndirectClient(final String name, final RedirectAction redirectAction
     @Override
     protected void clientInit() {
         defaultRedirectActionBuilder(ctx -> redirectAction);
-        defaultCredentialsExtractor(ctx -> returnCredentials.get());
+        defaultCredentialsExtractor(ctx -> Optional.ofNullable(returnCredentials.get()));
```

Comment:
ofNullable of a get on an optional? :)

Reply:
`returnCredentials` is of type `ReturnCredentials`, which is an interface that has only the `get()` method